lxd/sys/os.go

```diff
@@ -102,10 +102,13 @@ type OS struct {
 	LXCFeatures map[string]bool
 
 	// OS info
-	ReleaseInfo   map[string]string
-	KernelVersion version.DottedVersion
-	Uname         *shared.Utsname
-	BootTime      time.Time
+	ReleaseInfo map[string]string
+	Uname       *shared.Utsname
+	BootTime    time.Time
+
+	// Version info
+	KernelVersion   version.DottedVersion
+	AppArmorVersion *version.DottedVersion
```

Comment:
There's an AppArmor section of this struct this seems to make sense to go into. Also why is this a pointer? Is this so it can be optional? Can we make it 0 if it's not present, there's already an AppArmorAvailable field in this struct too.

Reply:
As this is internal struct its fine as is

Reply:
I went for a pointer as that's what `version.Parse()` returns and it's used in `appArmorGetVersion()`.

I can change it if you'd like.